src/main/java/core/basesyntax/Splitter.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Splitter {
+    public String[] split(String str) {
```

Comment:
this logic has become too specific

I would recommend to move this into a private helper method in ConsoleHandler class